src/Neo/SmartContract/Manifest/ContractAbi.cs

```diff
@@ -62,8 +62,8 @@ public static ContractAbi FromJson(JObject json)
         {
             ContractAbi abi = new()
             {
-                Methods = ((JArray)json["methods"]).Select(u => ContractMethodDescriptor.FromJson((JObject)u)).ToArray(),
-                Events = ((JArray)json["events"]).Select(u => ContractEventDescriptor.FromJson((JObject)u)).ToArray()
+                Methods = ((JArray)json!["methods"])!.Select(u => ContractMethodDescriptor.FromJson((JObject)u)).ToArray(),
+                Events = ((JArray)json!["events"])!.Select(u => ContractEventDescriptor.FromJson((JObject)u)).ToArray()
```

Comment:
```suggestion
                Methods = ((JArray)json!["methods"])?.Select(u => ContractMethodDescriptor.FromJson((JObject)u)).ToArray() ?? [],
                Events = ((JArray)json!["events"])?.Select(u => ContractEventDescriptor.FromJson((JObject)u)).ToArray() ?? []
```

You dont use `TryCatch` here. So I fixed it.